src/lib/createTreeExported.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * FIXME: THIS FILE IS A WORKAROUND TO CONFLICTING IMPORT STYLES.
```

Comment:
Could you elaborate a bit here, and say the file is duplicated because it is needed in both node.js (common.js) and web (ESM) environment?